pkg/chart/chart.go

```diff
@@ -328,6 +325,22 @@ func (c *Chart) InitNetClient(details *Details) (HTTPClient, error) {
 		}
 	}
 
+	defaultHeaders := http.Header{"User-Agent": []string{c.userAgent}}
+	if details.Auth.Header != nil {
+		namespace := os.Getenv("POD_NAMESPACE")
+		// TODO: Check with Andres: why are we relying on the defaultNamespace here for
+		// Auth, but not above for the customCA? Should both or neither?
+		// if namespace == "" {
+		// 	namespace = defaultNamespace
+		// }
```

Comment:
@andresmgot ^^

Reply:
yes, we should do it in both (this bug has never bitten us because we always set that env var in the chart

Reply:
Done - since both are in the one function, I moved them out so the namespace is set once and used for both.